code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -63,6 +63,7 @@
 	var/radiation_mod = 1                    // Radiation modifier
 	var/flash_mod =     1                    // Stun from blindness modifier.
 	var/vision_flags = SEE_SELF              // Same flags as glasses.
+	var/breakcuffs = list()                      //used in resist.dm to check if they can break hand/leg cuffs
```

Comment:
Change to `var/list/breakcuffs = list()`.

Otherwise the game will get confused occasionally.